README.txt

```diff
@@ -1,114 +1,84 @@
-=== Plugin Name ===
-Contributors: (this should be a list of wordpress.org userid's)
-Donate link: https://www.linknacional.com/
-Tags: comments, spam
-Requires at least: 3.0.1
-Tested up to: 3.4
-Stable tag: 4.3
-License: GPLv2 or later
-License URI: http://www.gnu.org/licenses/gpl-2.0.html
-
-Here is a short description of the plugin.  This should be no more than 150 characters.  No markup here.
+=== Payment Banking Slip Pix for LifterLMS ===
+Contributors: linknacional
+Donate link: https://www.linknacional.com/wordpress/plugins/
+Tags: lifterlms, bank, slip, pix, payment, banking.
+Requires at least: 5.5
+Tested up to: 6.2
+Stable tag: 1.0.0
+Requires PHP: 7.2
+License: GPLv3 or later
+License URI: https://www.gnu.org/licenses/gpl-3.0.html
+
+Enable bank slip and pix payment for LifterLMS..
 
 == Description ==
 
-This is the long description.  No limit, and you can use Markdown (as well as in the following sections).
+The [Payment Banking Slip Pix for LifterLMS](https://www.linknacional.com/wordpress/plugins/) is an extension plugin for LifterLMS which enables the bank slip and pix payment.
 
-For backwards compatibility, if this section is missing, the full length of the short description will be used, and
-Markdown parsed.
 
-A few notes about the sections above:
+**Dependencies**
 
-*   "Contributors" is a comma separated list of wp.org/wp-plugins.org usernames
-*   "Tags" is a comma separated list of tags that apply to the plugin
-*   "Requires at least" is the lowest version that the plugin will work on
-*   "Tested up to" is the highest version that you've *successfully used to test the plugin*. Note that it might work on
-higher versions... this is just the highest one you've verified.
-*   Stable tag should indicate the Subversion "tag" of the latest stable version, or "trunk," if you use `/trunk/` for
-stable.
+Payment Banking Slip Pix for LifterLMS plugin is dependent on LifterLMS plugin, please make sure LifterLMS is installed and properly configured before starting Payment Banking Slip Pix for LifterLMS installation.
```

Comment:
@FabricioLinkNacional Could you put a link to the Lifter LMS plugin? In the "[..] on LifterLMS plugin, [..]" put a markdown link to here: https://wordpress.org/plugins/lifterlms/